### PR TITLE
Fix a typo in a code example in a comment

### PR DESCRIPTION
**Pull request by yamt**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/storage/interfaces.go [hunk 1]
```diff
@@ -211,7 +211,7 @@ type Interface interface {
 	//       // a uint64 to alter the TTL on the object, or nil to keep it the same value.
 	//       return cur, nil, nil
 	//    }
```
Comment (mbohlool) [inline staging/src/k8s.io/apiserver/pkg/storage/interfaces.go]: it needs a "," here

Comment (yamt) [reply staging/src/k8s.io/apiserver/pkg/storage/interfaces.go]: why?

Comment (mbohlool) [reply staging/src/k8s.io/apiserver/pkg/storage/interfaces.go]: it is go syntax I guess when you close function parentheses on the next line, all your lines need to have ,.

[this](https://play.golang.org/p/U7W5eYvJ1JL) vs [this](https://play.golang.org/p/3ANYaBuHMWT)

Comment (yamt) [reply staging/src/k8s.io/apiserver/pkg/storage/interfaces.go]: thank you

**Pull request merged**